### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by toireasam**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ Creating and starting containers:
 ./ccd compose up -d
 ```
 
+You must run this command once.
```
Comment (geek-soft) [inline README.md]: Seems like wrong order. Modules should be enabled before first `./ccd compose up -d` is executed.

Comment (toireasam) [reply README.md]: you are correct :) i meant to add it before the compose up! will change now

File: README.md [hunk 2]
```diff
@@ -19,6 +19,11 @@ git submodule init
 git submodule update
 ```
 
+You must run this command once to enable modules.
+```
+./ccd enable backend frontend  sidam sidam-local sidam-local-ccd dm-store
```
Comment (geek-soft) [inline README.md]: You have two spaces but not a big deal if you do not want to clean that up.

Comment (toireasam) [reply README.md]: i will fix indeed :)

**Pull request merged**